salt/addons/init.sls

```diff
@@ -35,10 +35,11 @@ deploy_addons.sh:
   http.wait_for_successful_query:
```

Comment:
Maybe we should move this `wait_for_successful_query` to `kube-apiserver/init.sls:kube-apiserver`, so anyone needing the API server could just `require: service: kube-apiserver`.